package.json

```diff
@@ -34,8 +34,8 @@
     "prebuild": "run-s clean test",
     "build": "run-s clean esbuild:prod",
     "start": "run-s clean esbuild:dev",
-    "start-assisted-ui": "env-cmd ./scripts/start-assisted-ui.sh",
-    "start-uhc-portal": "./scripts/start-uhc-portal.sh",
+    "start-assisted-ui": "yarn --cwd ../assisted-ui start",
+    "start-uhc-portal": "yarn --cwd ../uhc-portal start",
     "sync-to-ui": "./scripts/sync-dist.sh",
```

Comment:
```suggestion
    "sync-dist": "./scripts/sync-dist.sh",
```

Reply:
Since it is no longer just synchronizing the dist folder with the `assisted-ui`...